src/include/ipc/topology.h

```diff
@@ -67,8 +67,7 @@ struct sof_ipc_comp {
 	uint32_t pipeline_id;
 	uint32_t core;
 
-	/* reserved for future use */
-	uint32_t reserved[1];
+	uint32_t subtype;       /**< favour for generic component type */
```

Comment:
This should be using UUID now. There's no pint in introducing a short lived ABI change.

Reply:
@lgirdwood as I stated above, the problem is that the 16Bytes UUID will not be accepted (Firmware can't understand it) by today's firmware.

Reply:
If it needs a new IPC then we add a new IPC. This subtype is just the old method.